modules/@apostrophecms/migration/index.js

```diff
@@ -245,6 +245,15 @@ module.exports = {
               await self.runOne(migration);
             }
           }
+          // In production, this event is emitted only at the end of the migrate command line task.
+          // In dev it is emitted at every startup after the automatic migration.
+          //
+          // Intentionally emitted regardless of whether the site is new or not.
+          //
+          // This is the right time to park pages, for instance, because the
+          // database is guaranteed to be in a sane state, whether because the
+          // site is new or because migrations ran successfully.
+          await self.emit('after');
```

Comment:
In late A2, we introduced a top-level `apostrophe:migrate` event for the same reason. However I think `@apostrophecms/migrate:after` is better because migrations are managed by that module and things that can be managed by modules in Apostrophe should be. The other top level events are legitimately top level because they are things like `modulesReady` and `afterInit` that are defined by *all* modules having reached a certain point.